tests/Platforms/SqlitePlatformTest.php

```diff
@@ -768,4 +771,19 @@ public function testGetCreateTableSQLWithColumnCollation(): void
             $this->platform->getCreateTableSQL($table),
         );
     }
+
+    public function testCreateTableInOtherSchema(): void
+    {
+        $table = new Table('other_schema.foo');
+        $table->addColumn('id', Types::INTEGER);
+        $table->addIndex(['id']);
+
+        self::assertSame(
+            [
+                'CREATE TABLE other_schema.foo (id INTEGER NOT NULL)',
+                'CREATE INDEX other_schema.IDX_D9A8AD0DBF396750 ON foo (id)',
```

Comment:
I would not introduce this test. We do not care what the resulting SQL looks like as long as the functionality is covered by a test. Furthermore, this test is definitely not the source of truth about what the index name should be.

Reply:
The problem is that the functional test would pass if the platform decided to create no index at all.

I agree about the generated name though. If we make assertions on the name if the index, we should explicitly declare a name for the index.

Reply:
Should I remove the unit test and modify the functional test to assert that an index does exist? If the platform decides not to create it, then introspecting the table won't show the index.

Reply:
Actually, I stand correcting. This fails for some reason:

```
        $onlineTable = $this->connection->getSchemaManager()->listTableDetails('other.test_other_schema');
        self::assertTrue($onlineTable->hasIndex('id'));
```

Reply:
If we need to ensure that the index exists, we can introspect the schema.

Reply:
Yep I think I did exactly that:)